policyuniverse/expander_minimizer.py

```diff
@@ -171,17 +172,15 @@ def minimize_statement_actions(statement, minchars=None):
 def get_actions_from_statement(statement):
     allowed_actions = set()
 
-    if not type(statement.get("Action", [])) == list:
-        statement["Action"] = [statement["Action"]]
+    statement["Action"] = ensure_array(statement.get("Action", []))
 
-    for action in statement.get("Action", []):
+    for action in statement["Action"]:
```

Comment:
Won't this fail and throw an exception if the statement uses `NotAction` instead of `Action` ?

Reply:
I don't think it would, because we had set this above. However, I thought that we could also avoid mutation. I provided an alternative in 998b50b94598fdd99933db597fc4d89dcff66cb9 , let me know what you think!

Reply:
LGTM